samples/subsys/usb/cdc_acm/prj.conf

```diff
@@ -4,7 +4,7 @@ CONFIG_GPIO=y
 CONFIG_USB=y
 CONFIG_USB_DEVICE_STACK=y
 CONFIG_USB_DEVICE_PRODUCT="Zephyr CDC ACM sample"
-CONFIG_SYS_LOG_USB_DRIVER_LEVEL=1
+CONFIG_USB_DRIVER_LOG_LEVEL=1
```

Comment:
Please add CONFIG_LOG=y as well to all prj*.conf

Reply:
Done